app/Data/page.ts

```diff
@@ -0,0 +1,46 @@
+"use client";
+
+export interface Contacts {
```

Comment:
Interface should be singular, if you want multiple of a single contact, then you do:
const contacts: Contact[] = []....